user_guide.md

```diff
@@ -86,6 +86,12 @@ Now let's set `${KF_ENV}` to `cloud` or `nocloud` to reflect our environment for
 $ KF_ENV=cloud|nocloud
 ```
 
+By default Kubeflow does not persist any work that is done within jupyter notebook. That means if container is destroyed or recreated, all it's contents, including users working notebooks etc. are going to be deleted. To enable persistence user will first need default will need to have default storage class for [persistent volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/). If that's the case, persistency can be enabled by setting jupyterNotebookPVCMount.
```

Comment:
"will first need default will need to have default storage class"
There is some duplication there.